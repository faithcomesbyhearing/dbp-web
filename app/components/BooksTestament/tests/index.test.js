// BooksTestament.test.js

import React from 'react';
import { render } from '@testing-library/react';
import { fromJS } from 'immutable';
import BooksTestament from '..';

const books = fromJS([
	{
		book_id: 'GEN',
		book_id_usfx: 'GN',
		book_id_osis: 'Gen',
		name: 'Genesis',
		testament: 'OT',
		testament_order: 1,
		book_order: 1,
		book_group: 'The Law',
		chapters: Array.from({ length: 50 }, (_, i) => i + 1),
	},
	{
		book_id: 'EXO',
		book_id_usfx: 'EX',
		book_id_osis: 'Exod',
		name: 'Exodus',
		testament: 'OT',
		testament_order: 2,
		book_order: 2,
		book_group: 'The Law',
		chapters: Array.from({ length: 40 }, (_, i) => i + 1),
	},
	{
		book_id: 'LEV',
		book_id_usfx: 'LV',
		book_id_osis: 'Lev',
		name: 'Leviticus',
		testament: 'OT',
		testament_order: 3,
		book_order: 3,
		book_group: 'The Law',
		chapters: Array.from({ length: 27 }, (_, i) => i + 1),
	},
]);

const handleRef = jest.fn();
const handleBookClick = jest.fn();
const handleChapterClick = jest.fn();
const activeChapter = 1;
const testamentPrefix = 'ot';
const testamentTitle = 'Old Testament';
const selectedBookName = 'Genesis';
const activeBookName = 'Genesis';
const activeTextId = 'ENGESV';
const audioType = 'drama';

describe('<BooksTestament />', () => {
	it('Should match the old snapshot', () => {
		const { asFragment } = render(
		<BooksTestament
			books={books}
			handleRef={handleRef}
			handleBookClick={handleBookClick}
			handleChapterClick={handleChapterClick}
			activeChapter={activeChapter}
			testamentPrefix={testamentPrefix}
			testamentTitle={testamentTitle}
			selectedBookName={selectedBookName}
			activeBookName={activeBookName}
			activeTextId={activeTextId}
			audioType={audioType}
		/>,
		);
		expect(asFragment()).toMatchSnapshot();
	});

	it('Should only render one active book', () => {
		const { container } = render(
		<BooksTestament
			books={books}
			handleRef={handleRef}
			handleBookClick={handleBookClick}
			handleChapterClick={handleChapterClick}
			activeChapter={activeChapter}
			testamentPrefix={testamentPrefix}
			testamentTitle={testamentTitle}
			selectedBookName={selectedBookName}
			activeBookName={activeBookName}
			activeTextId={activeTextId}
			audioType={audioType}
		/>,
		);

		// Query all elements with the 'active-book' class
		const activeBooks = container.querySelectorAll('.active-book');
		expect(activeBooks.length).toBe(1);
	});

	it('Should only render one active chapter and it should match the given prop', () => {
		const { container } = render(
		<BooksTestament
			books={books}
			handleRef={handleRef}
			handleBookClick={handleBookClick}
			handleChapterClick={handleChapterClick}
			activeChapter={activeChapter}
			testamentPrefix={testamentPrefix}
			testamentTitle={testamentTitle}
			selectedBookName={selectedBookName}
			activeBookName={activeBookName}
			activeTextId={activeTextId}
			audioType={audioType}
		/>,
		);

		// Query all elements with the 'active-chapter' class
		const activeChapters = container.querySelectorAll('.active-chapter');
		expect(activeChapters.length).toBe(1);
		expect(activeChapters[0].textContent).toBe('1');
	});
});
