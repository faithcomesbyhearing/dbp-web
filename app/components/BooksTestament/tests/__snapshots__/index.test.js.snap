// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BooksTestament /> Should match the old snapshot 1`] = `
[
  <div
    className="testament-title"
  >
    <h3>
      Old Testament
    </h3>
  </div>,
  <div
    className="book-button"
    id="GenesisGEN"
    onClick={[Function]}
  >
    <h4
      className="active-book"
    >
      Genesis
    </h4>
    <span
      className="chapter-container active-book-chapters"
    >
      <a
        className="chapter-box"
        onClick={[MockFunction]}
      >
        <span
          className="active-chapter"
        >
          1
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/2?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          2
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/3?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          3
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/4?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          4
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/5?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          5
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/6?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          6
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/7?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          7
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/8?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          8
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/9?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          9
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/10?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          10
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/11?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          11
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/12?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          12
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/13?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          13
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/14?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          14
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/15?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          15
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/16?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          16
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/17?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          17
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/18?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          18
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/19?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          19
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/20?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          20
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/21?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          21
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/22?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          22
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/23?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          23
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/24?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          24
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/25?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          25
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/26?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          26
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/27?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          27
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/28?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          28
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/29?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          29
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/30?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          30
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/31?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          31
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/32?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          32
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/33?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          33
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/34?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          34
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/35?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          35
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/36?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          36
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/37?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          37
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/38?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          38
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/39?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          39
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/40?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          40
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/41?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          41
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/42?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          42
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/43?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          43
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/44?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          44
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/45?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          45
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/46?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          46
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/47?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          47
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/48?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          48
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/49?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          49
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/GEN/50?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          50
        </span>
      </a>
    </span>
  </div>,
  <div
    className="book-button"
    id="ExodusEXO"
    onClick={[Function]}
  >
    <h4
      className=""
    >
      Exodus
    </h4>
    <span
      className="chapter-container inactive-book-chapters"
    >
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/1?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          1
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/2?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          2
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/3?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          3
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/4?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          4
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/5?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          5
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/6?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          6
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/7?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          7
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/8?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          8
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/9?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          9
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/10?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          10
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/11?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          11
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/12?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          12
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/13?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          13
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/14?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          14
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/15?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          15
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/16?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          16
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/17?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          17
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/18?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          18
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/19?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          19
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/20?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          20
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/21?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          21
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/22?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          22
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/23?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          23
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/24?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          24
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/25?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          25
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/26?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          26
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/27?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          27
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/28?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          28
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/29?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          29
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/30?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          30
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/31?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          31
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/32?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          32
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/33?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          33
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/34?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          34
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/35?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          35
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/36?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          36
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/37?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          37
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/38?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          38
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/39?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          39
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/EXO/40?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          40
        </span>
      </a>
    </span>
  </div>,
  <div
    className="book-button"
    id="LeviticusLEV"
    onClick={[Function]}
  >
    <h4
      className=""
    >
      Leviticus
    </h4>
    <span
      className="chapter-container inactive-book-chapters"
    >
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/1?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          1
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/2?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          2
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/3?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          3
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/4?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          4
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/5?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          5
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/6?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          6
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/7?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          7
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/8?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          8
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/9?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          9
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/10?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          10
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/11?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          11
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/12?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          12
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/13?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          13
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/14?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          14
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/15?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          15
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/16?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          16
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/17?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          17
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/18?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          18
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/19?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          19
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/20?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          20
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/21?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          21
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/22?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          22
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/23?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          23
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/24?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          24
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/25?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          25
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/26?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          26
        </span>
      </a>
      <a
        className="chapter-box"
        href="/bible/ENGESV/LEV/27?audio_type=drama"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <span>
          27
        </span>
      </a>
    </span>
  </div>,
]
`;
